src/Object/Product/Product.php

```diff
@@ -869,4 +878,32 @@ public function setThumbUrl($thumbUrl)
     {
         $this->thumbUrl = $thumbUrl;
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function getVariations()
```

Comment:
Good point, how should we deal with this? Just call it variations or add support to Nosto's end to handle `price_variations`?